pyvista/core/filters/image_data.py

```diff
@@ -1038,6 +1039,10 @@ def contour_labels(  # type: ignore[misc]
             generated between foreground background. ``'all'``  includes both internal
             and external boundary polygons.
 
+        background_value : int, default: 0
+            Background value of the input image. All other values are considered
+            as foreground.
+
```

Comment:
```suggestion
            .. versionadded:: 0.45.0

```

Reply:
This filter is not yet released, so all parameters are new to 0.45. Versionadded is used in the main description, is that sufficient or it's better to also include it with the parameters?

Reply:
Oops didn't realised that. Thanks!